graph-db/serverless/src/utils/cmr/__test__/fetchCollectionPermittedGroups.test.js

```diff
@@ -0,0 +1,183 @@
+import nock from 'nock'
+
+import { fetchCollectionPermittedGroups } from '../fetchCollectionPermittedGroups'
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('fetchCollectionPermittedGroups', () => {
+  test('returns collection permitted groups metadata', async () => {
+    const mockedBody = {
+      items: [
+        {
+          concept_id: 'ACL1376510432-CMR',
+          revision_id: 9,
+          identity_type: 'Catalog Item',
+          acl: {
+            group_permissions: [{ permissions: ['read'], group_id: 'AG1337430038-NSIDC_ECS' },
+              { permissions: ['read'], group_id: 'AG1337430039-NSIDC_ECS' },
+              { permissions: ['read'], group_id: 'AG1337409405-CMR' },
+              { permissions: ['read'], group_id: 'AG1386452130-CMR' },
+              { permissions: ['read'], group_id: 'AG1337409406-CMR' },
+              { permissions: ['read'], user_type: 'registered' },
+              { permissions: ['read'], group_id: 'AG1337409407-CMR' }],
+            catalog_item_identity: [Object],
+            legacy_guid: '26B6710B-0562-953D-CCE7-E185B36A9545'
+          },
+          name: 'All Collections',
+          location: 'https://cmr.earthdata.nasa.gov:443/access-control/acls/ACL1376510432-CMR'
+        },
+        {
+          concept_id: 'ACL1374052769-CMR',
+          revision_id: 81,
+          identity_type: 'Catalog Item',
+          acl: {
+            group_permissions: [
+              { permissions: ['read', 'order'], group_id: 'AG1337430038-NSIDC_ECS' },
+              { permissions: ['read', 'order'], group_id: 'AG1337409405-CMR' },
+              { permissions: ['read', 'order'], group_id: 'AG1337409406-CMR' },
+              { permissions: ['read', 'order'], group_id: 'AG1337409413-CMR' },
+              { permissions: ['read', 'order'], user_type: 'guest' },
+              { permissions: ['read'], user_type: 'guest' },
+              { permissions: ['read', 'order'], group_id: 'AG1337409411-CMR' }],
+            catalog_item_identity: [Object],
```

Comment:
This `Object` is just a product of whatever tool this was output to not rendering that deep, this should contain data or just be an empty array.

Reply:
Fixed